main.js

```diff
@@ -177,6 +178,22 @@ document.getElementById("start").onclick = function() {
   start();
 };
 
+document.getElementById("canvasSize").onchange = function() {
+  Array.prototype.forEach.call(document.body.classList, function(c) {
+    if (c.indexOf('size-') == 0) {
+      document.body.classList.remove(c);
+    }
+  });
+
+  if (this.value) {
+    document.body.classList.add(this.value);
+  }
+
+  MIDP.updatePhysicalScreenSize();
+  MIDP.updateCanvas();
+  start();
```

Comment:
This doesn't quite do what I would expect, as the display div does change size, but the canvas element within it doesn't do the same, when I pick a new size while running tests.